openmc/region.py

```diff
@@ -46,6 +46,27 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def update_surfaces(self, surfaces=None):
+        """
+        Recursively find all the surfaces referenced by a region and return them
+
+        Parameters
+        ----------
+        surfaces: collections.OrderedDict, optional
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
+            
+        Returns
+        -------
+        surfaces: collections.OrderedDict
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
+
+        """
+        if not surfaces:
```

Comment:
`if surfaces is not None` would be a better check. If someone passed an empty dictionary, it actually wouldn't be updated.